awesomplete.js

```diff
@@ -301,9 +309,7 @@ function $$(expr, con) {
 	return slice.call((con || document).querySelectorAll(expr));
 }
 
-$.create = function(tag, o) {
-	var element = document.createElement(tag);
-
+$.set = function(element, o) {
```

Comment:
Definitely +1 for modularization, might be a good time to take care off npm stuff as well. The build/release process can, and I think should be automated, so once done it won't take any time. Can send a PR, just let me know.